eng/connection-strings.psm1

```diff
@@ -29,10 +29,10 @@ function Build-PostgresConnectionString {
     $connectionString = "host=$Server;port=$Port;database=$DatabaseName;"
 
     if ($UseIntegratedSecurity) {
-        return $connectionString + "Integrated Security=true"
+        return $connectionString + "Integrated Security=true;Encrypt=False;"
     }
 
-    return $connectionString + "User Id=$Username;Password=$Password"
+    return $connectionString + "User Id=$Username;Password=$Password;Encrypt=False;"

```

Comment:
How is this connection string being used? Might be better to make this a user-generated parameter instead of hard-coding this value.

Reply:
Agree with @stephenfuqua. We documented the details. No need to hard code.  Admin API 2.x installer accepting the "UnEncryptedConnection" parameter from user, during installation.

Reply:
Agree with @stephenfuqua. We have documentation. On Admin API 2.x installer we are accepting user parameter "UnEncryptedConnection" to take decision.

Reply:
I made this change while troubleshooting something else. This connection string is only being used by integration tests so I will make this change in the backlog ticket to fix/improve integration tests.